### PR TITLE
Skip auditing WLambda

### PR DESCRIPTION
**Pull request by jacob-hughes**

### CODE REVIEW

File: audit.py [hunk 1]
```diff
@@ -30,6 +30,7 @@
     # unmaintained repos.
     ("softdevteam", "k2"),
     ("softdevteam", "error_recovery_experiment"),
+    ("softdevteam", "WLambda"),
```
Comment (vext01) [inline audit.py:33]: Is WLambda unmaintained then?

Comment (ltratt) [reply audit.py:33]: For us it's a benchmarking target, so we'll either get those changes from upstream or... not.

Comment (vext01) [reply audit.py:33]: I think we should add a new section comment like:

```
# maintained externally
```

or similar.

Comment (jacob-hughes) [reply audit.py:34]: Good idea. Fixed here ad7ff8eea4745d5fb6adb7b6579929f01e42710f

**Comment by vext01** [comment]
please squash!

**Comment by jacob-hughes** [comment]
Squashed

**Pull request merged**